### ISSUE TITLE
Low accuracy with landmark detection task

### ISSUE THREAD

**Issue by karaposu** [Kartik-3004/facexformer#13]
When i try to hf model with example images it shows bad accuracy in certain facial areas (eyes specially). Here is an example:

<img width="427" alt="Screenshot 2024-05-14 at 12 12 38" src="https://github.com/Kartik-3004/facexformer/assets/31979644/5ed480be-0bd1-4df9-888e-8c2ab4c13e2e">

**Comment by uozyurt** [comment]
Actually, the main reason for that is the inputs alignement. You have to properly crop the face if you want to get accurate outputs in these kind of tasks (specifically, facial attribute recognition and landmark detection).

The main mistake is you have too much space above the input face. If you cut this space off, it will give you accurate results (I tried to input in both ways in my local for getting the problem you had). Hope that I fixed your problem!

**Comment by karaposu** [comment]
@uozyurt Yeah makes sense. This model is a bit extra sensitive about cropping compared to similar ones. Thanks umut
